.rhiza/tests/integration/test_sbom.py

```
@@ -0,0 +1,172 @@
+            "uvx",
+            "--from",
+            "cyclonedx-bom>=7.0.0",
+            "cyclonedx-py",
+            "environment",
+            "--pyproject",
+            "pyproject.toml",
+            "--of",
+            "JSON",
+            "-o",
+            "sbom.cdx.json",
+        ],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)